### DISCUSSION TITLE
Quick question: Is ibis.util.promote_list() a stable API?

### DISCUSSION THREAD

**Discussion by NickCrews** [ibis-project/ibis#6700] [Q&A] [answered]
Can I use this in my application code, or could this change in the future? Is it a similar story for all non _private stuff in ibis.util? Thanks!

**Comment by cpcloud** [comment, marked as answer]
Per the new [versioning policy](https://ibis-project.org/versioning/) it should not be considered stable.

**Reply by NickCrews** [reply]
Sounds good. Thanks for the explicit policy!
